src/script/engine.zig

```diff
@@ -219,6 +204,15 @@ pub const Engine = struct {
         try self.stack.pushByteArray(&[_]u8{0x81});
     }
 
+    /// OP_RESERVED: Reserved opcode
+    pub fn opReserved(self: *Engine, opcode: u8) !void {
```

Comment:
Please expose one method by opcode. If you want to factorize code, do it internally, not i the public methods